app/src/main/java/com/example/lhi06/residue/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+package com.example.lhi06.residue;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+
+public class MainActivity extends AppCompatActivity {
+
+    @BindView(R.id.add_review)
+
+    private Button searchButton;
+    private EditText editText1;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        editText1 = (EditText) findViewById(R.id.request);
+        searchButton = (Button) findViewById(R.id.searchButton);
+        FloatingActionButton addReviewButton = (FloatingActionButton) findViewById(R.id.add_review);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        searchButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                // Code for searching for reviews done
+//                String artistName = editText1.getText().toString();
+//                handler.urlBuilder(artistName);
+            }
+        });
+
+        addReviewButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+            //will navigate to the album searching activity
```

Comment:
Can you add TODO to the start of this comment please
//TODO: Navigate to the album searching activity